export/codeSignGroup.go

```diff
@@ -0,0 +1,13 @@
+package export
```

Comment:
do not use camelCase file name, eighter name it: `codesign_group.go` or `group.go`